src/engine/Util/Detector.ts

```diff
@@ -1,13 +1,134 @@
 /// <reference path="Log.ts" />
+
+/**
+ * This is the list of features that will be used to log the supported
+ * features to the console when Detector.logBrowserFeatures() is called.
+ */
+const REPORTED_FEATURES: Object = {
+   webgl: 'WebGL',
+   webaudio: 'WebAudio',
+   gamepadapi: 'Gamepad API'
+};
+
 module ex {
 
    /**
+    * Interface for detected browser features matrix
+    */
+   export interface IDetectedFeatures {
+      canvas: boolean;
```

Comment:
Optional: could add the new `readonly` modifier to each of the properties so TS will throw if someone accidentally assigns to these props.